test/new-e2e/README.md

```diff
@@ -29,7 +29,7 @@ As a workaround, you can run tests against the version of the agent built in an
 
 ### Installer tests
 
-Installer tests are tests owned by Agent Platform, that mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
+Installer tests are tests owned by Agent Build And Releases. They mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
```

Comment:
Oh we own those? TIL :sweat_smile:

Reply:
I don't think it's unreasonable for us to own these 🤷